README.md

```diff
@@ -30,6 +30,7 @@ Now with Github deploys!
       API_BASE_URL=<franklin_api_url>
       SOCIAL_AUTH_GITHUB_KEY=<github_client_id>
       SOCIAL_AUTH_GITHUB_SECRET=<github_client_secret>
+      GITHUB_SECRET=<for_validating_github_webhook_messages>
```

Comment:
until I can fix it, you'll also need a `GITHUB_OAUTH` with your personal oauth token so we can call github. social signin token not saving correctly right now.